client/my-sites/posts/main.jsx

```diff
@@ -73,14 +80,19 @@ const PostsMain = React.createClass( {
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ this.translate( 'Latest Drafts' ) }>
```

Comment:
Drafts won't get rendered unless there are drafts available, as determined in [`showDrafts()`](https://github.com/Automattic/wp-calypso/blob/c1c823718240ed99165e637fcecceb4883fd7b2c/client/my-sites/posts/main.jsx#L59)